cinema/views.py

```diff
@@ -0,0 +1,42 @@
+from django.http import JsonResponse, HttpResponse
+from django.views.decorators.csrf import csrf_exempt
+from django.shortcuts import get_object_or_404
+from rest_framework.parsers import JSONParser
+from .models import Movie
+from .serializers import MovieSerializer
+
+@csrf_exempt
+def movie_list(request):
+    if request.method == 'GET':
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return JsonResponse(serializer.data, safe=False)
+
+    elif request.method == 'POST':
+        data = JSONParser().parse(request)
+        serializer = MovieSerializer(data=data)
+        if serializer.is_valid():
+            serializer.save()
+            return JsonResponse(serializer.data, status=201)
+        return JsonResponse(serializer.errors, status=400)
+
+
+@csrf_exempt
```

Comment:
Similar to the `movie_list` view, using `@csrf_exempt` here is not recommended. Use `@api_view` from Django REST Framework to ensure proper CSRF protection.